src/renderer/services/file-storage-service/index.ts

```diff
@@ -111,7 +112,8 @@ export default class FileStorageService extends HttpCacheClient {
       // so the conversion must be manual each request
       file_size: fileSize,
       local_nas_shortcut: localNasPath !== undefined,
-      local_nas_path: localNasPath
+      local_nas_path: localNasPath,
+      multifile: isMultifile !== undefined ? isMultifile : false,
```

Comment:
```suggestion
      multifile :!!isMultifile,
```

Reply:
I'm usually more in favor of verbosity over cool code things, but JS isn't my bag so it isn't an informed opinion in this case.

Reply:
I would call this 'Matty-esque'

Reply:
I say we try it out

Reply:
changed in 9d9d665b6c4bd85c1f53ba4c5a5af41a5eba2168